webui/src/views/categories/dialogs/CategoryDialog.vue

```diff
@@ -176,10 +80,10 @@ const cancel = () => {
     emit('update:visible', false)
 }
 
-// Show parent selector condition
-const showParentSelector = computed(() => {
-    return localCategory.value && localCategory.value.action !== 'add'
-})
+// Show parent selector only when editing
+// const showParentSelector = computed(() => {
```

Comment:
leftover to be enabled ?
we should only see the root node when editing categories, but not when assigning entries to categories